testsuite/features/core_ubuntu_salt.feature

```diff
@@ -57,16 +64,16 @@ Feature: Be able to bootstrap an Ubuntu minion and do some basic operations on i
 
 @ubuntu_minion
   Scenario: Run a remote command on the Ubuntu minion
```

Comment:
Same, this kind of tests is now done in the secondary features (we want core to be leightweight).

Reply:
Totally agree. But we will do in a separate PR.